contracts/exchange/api/MarginConfigContract.sol

```diff
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.20;
```

Comment:
is this technically accurate? Since I don't see an entry for it here https://spdx.org/licenses/ so it might be misunderstood. 

Another approach is to put in a license file at the top level and say all rights reserved for example?